src/display/LiquidCrystalAdapter.h

```diff
@@ -26,6 +26,10 @@ class LiquidCrystalAdapter : public DisplayInterface {
         lcd->clear();
         lcd->createChar(0, upArrow);
         lcd->createChar(1, downArrow);
+        byte celsius[8] = {B11100, B10100, B11100, B0000, B00000, B00000, B00000, B00000};
+        byte plusMinus[] = {B00000, B00100, B01110, B00100, B00000, B01110, B00000, B00000};
+        lcd->createChar(2, celsius);
+        lcd->createChar(3, plusMinus);
```

Comment:
These can be created outside the display adapter (in `setup()`) as you have the lcd instance there, doesn't really need to be here.

Another option to make it maybe easy for use, is to extract them into another utility class eg `DisplayConfig`, in the `DisplayConfig` for LCD screens for example, one can create the custom chars outside and pass them into the display interface through the `DisplayConfig`. In the example sketch, you would pass `\002` and `\003` and they'll be created. We can also use it for customising the indicator arrows and the other custom chars that exist in the LCD adapters. The option of using this class would be maybe better as the dev wouldn't know which bytes have already been taken unless the dive deep into the code.